haystack/backends/solr_backend.py

```diff
@@ -48,6 +55,14 @@ def __init__(self, connection_alias, **connection_options):
         self.conn = Solr(connection_options['URL'], timeout=self.timeout, **connection_options.get('KWARGS', {}))
         self.log = logging.getLogger('haystack')
 
+    def get_schema_admin(self):
```

Comment:
Never used it. Could be great but would break compatibility with Django 1.6 (may be it's already not compatible...).